updated DOC (#1014)
microsoftgraph/entra-powershell: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDirSyncfeature.md

---
title: Get-EntraBetaDirSyncFeature
description: This article provides details on the Get-EntraBetaDirSyncFeature command.

ms.topic: reference
ms.date: 08/16/2024
ms.author: eunicewaweru
ms.reviewer: stevemutungi
manager: CelesteDG
author: msewaweru

external help file: Microsoft.Graph.Entra.Beta-help.xml
Module Name: Microsoft.Graph.Entra.Beta
online version: https://learn.microsoft.com/powershell/module/Microsoft.Graph.Entra.Beta/Get-EntraBetaDirSyncFeature

schema: 2.0.0
---

# Get-EntraBetaDirSyncFeature

## Synopsis

> Used to check the status of identity synchronization features for a tenant.

Reviewer: ```suggestion
Checks the status of identity synchronization features for a tenant.
```

- **EnableSoftMatchOnUpn**
- **PasswordSync**
- **SynchronizeUpnForManagedUsers**
- **UnifiedGroupWriteback**
- **UserWriteback**

The cmdlet can be run without specifying any features, in which case it returns a list of all features and their enabled or disabled status.

## Examples

### Example 1:  Return a list of all possible DirSync features and whether they're enabled (True) or disabled (False)

```Powershell
Connect-Entra -Scopes 'OnPremDirectorySynchronization.Read.All'
Get-EntraBetaDirSyncFeature
```

```Output
Enabled DirSyncFeature
------- --------------
  False BlockCloudObjectTakeoverThroughHardMatch
  False BlockSoftMatch
  False BypassDirSyncOverrides
  False CloudPasswordPolicyForPasswordSyncedUsers
  False ConcurrentCredentialUpdate
   True ConcurrentOrgIdProvisioning
  False DeviceWriteback
  False DirectoryExtensions
  False FopeConflictResolution
  False GroupWriteBack
  False PasswordSync
  False PasswordWriteback
   True QuarantineUponProxyAddressesConflict
   True QuarantineUponUpnConflict
   True SoftMatchOnUpn
   True SynchronizeUpnForManagedUsers
  False UnifiedGroupWriteback
  False UserForcePasswordChangeOnLogon
  False UserWriteback
```

> This command returns a list of all possible DirSync features and whether they're enabled (True) or disabled (False).

Reviewer: ```suggestion
This example gets a list of all DirSync features and shows if they are enabled (True) or disabled (False).
```

### Example 2:  Return whether PasswordSync is enabled for the tenant (True) or disabled (False)

```powershell
Connect-Entra -Scopes 'OnPremDirectorySynchronization.Read.All'
Get-EntraBetaDirSyncFeature -Feature 'PasswordSync'
```

```Output
Enabled DirSyncFeature
------- --------------
  False PasswordSync
```

> This command returns whether PasswordSync is enabled for the tenant (True) or disabled (False).

Reviewer: ```suggestion
This example shows if PasswordSync is enabled (True) or disabled (False) for the tenant.
```

> - `Feature` Parameter specifies DirSync feature to get the status of.

Reviewer: ```suggestion
- `Feature` specifies the DirSync feature to check the status of.
```

## Parameters

> ### -TenantId

Reviewer: Should be a System.String instead of a GUID

```

### -Feature

> The DirSync feature to get the status of.

Reviewer: ```suggestion
The DirSync feature to check the status of.
```
